provisioner/roles/manage_ec2_instances/tasks/inventory/addhost_rhel.yml

```diff
@@ -38,3 +56,19 @@
     - "{{ node2_node_facts.instances }}"
     - "{{ node3_node_facts.instances }}"
   changed_when: false
+
+- name: add nodes for cluster lab to groups (ANSIBLE RHEL MODE)
+  add_host:
+    name: "{{ item.tags.Name }}"
+    username: "{{ item.tags.Student }}"
+    short_name: "{{ item.tags.short_name }}"
+    ansible_host: "{{ item.public_ip_address }}"
+    ansible_user: "{{ item.tags.username }}"
+    ansible_port: "{{ ssh_port }}"
+    ansible_ssh_private_key_file: "{{ playbook_dir }}/{{ec2_name_prefix}}/{{ec2_name_prefix}}-private.pem"
+    groups: lab_hosts,managed_nodes
+  with_items:
+    - "{{ isonode_node_facts.instances }}"
+    - "{{ remotenode_node_facts.instances }}"
+  #changed_when: false
```

Comment:
@goetzrieger Last piece: please add a space here after the comment sign, linter otherwise complains. Rest loks good, thanks.